app/assets/javascripts/misc/toggle-button.js

```diff
@@ -0,0 +1,24 @@
+import 'classlist.js';
+
+function toggleButton() {
+  const accordion = document.querySelector('.accordion');
+
+  if (accordion) {
+    const controls = accordion.querySelectorAll('[aria-controls]');
+    const button = document.querySelector('.js-toggle-button');
+
+    controls.forEach((control) => {
```

Comment:
I'm not sure... it does work though 🤔   I think Chrome may support iterating over it directly like that, but guessing not other browsers.